package.json

```diff
@@ -57,14 +57,14 @@
   "dependencies": {
     "@knapsack-pro/core": "^3.2.0",
     "glob": "^8.0.3",
+    "minimatch": "^5.1.2",
     "minimist": "^1.2.6",
     "uuid": "^8.3.2"
   },
   "peerDependencies": {
     "cypress": ">=10.0.0 <11.0.0"
   },
   "devDependencies": {
-    "cypress": "10.1.0",
```

Comment:
Did you accidentally remove it? Could you bring it back?

Reply:
Fixed